stacker_blueprints/sns.py

```diff
@@ -54,16 +83,48 @@ def create_template(self):
             self.create_topic(topic_name, topic_config)
 
     def create_topic(self, topic_name, topic_config):
+        subscriptions = False
         t = self.template
 
+        if "Subscription" in topic_config:
+            subscriptions = True
+            topic_subs = topic_config["Subscription"]
+            topic_config["Subscription"] = process_subs(topic_subs)
+
         t.add_resource(
             sns.Topic(
                 topic_name,
                 **topic_config
             )
         )
 
+        topic_arn = Ref(topic_name)
+
         t.add_output(
             Output(topic_name + "Name", Value=GetAtt(topic_name, "TopicName"))
         )
-        t.add_output(Output(topic_name + "Arn", Value=Ref(topic_name)))
+        t.add_output(Output(topic_name + "Arn", Value=topic_arn))
+
+        if subscriptions:
+            arn_endpoints = []
+            url_endpoints = []
+            for sub in topic_subs:
+                arn_endpoints.append(sub["Endpoint"])
+                split_endpoint = sub["Endpoint"].split(":")
+                queue_url = "https://%s.%s.amazonaws.com/%s/%s" % (
```

Comment:
i think using named variables in the string interpolation would make this easier to understand to, something like: `"https://%(region)..." % {"region": split_endpoint[1]}`